.example.env

```diff
@@ -0,0 +1,18 @@
+## Example environment file for docker-compose and builders
+
+COMPOSE_PROJECT_NAME=postgis
+
+## For build arguments
+DISTRO=debian
+IMAGE_VERSION=bullseye
+IMAGE_VARIANT=slim
+# Set GENERATE_ALL_LOCALE to empty value or 0 to build just default LOCALE: en_US.UTF-8
+GENERATE_ALL_LOCALE=1
+# Set the language if you need to specify LANG locale at build time
+LANG=en_US.UTF-8
+# locale filter to include in the locale generator
+LANGS="en_US.UTF-8,id_ID.UTF-8"
+
+POSTGRES_MAJOR_VERSION=13
+POSTGIS_MAJOR_VERSION=3
+POSTGIS_MINOR_RELEASE=1
```

Comment:
Build arguments (and runtime docker-compose arguments) must be declared here.

Local user can do custom local build by copying this template into `.env` and fill in their values.
When we have changes to this file, user can diff their `.env` to this file for reference.